torch/nn/modules/rnn.py

```diff
@@ -1029,15 +1029,18 @@ class LSTMCell(RNNCellBase):
         where :math:`k = \frac{1}{\text{hidden\_size}}`
 
     Examples::
+    
+    
 
-        >>> rnn = nn.LSTMCell(10, 20)
-        >>> input = torch.randn(3, 10)
-        >>> hx = torch.randn(3, 20)
-        >>> cx = torch.randn(3, 20)
+        >>> rnn = nn.LSTMCell(10, 20)        # (input_size, hidden_size)
+        >>> input = torch.randn(2, 3, 10)    # (time_steps, batch, input_size)
+        >>> hx = torch.randn(3, 20)          # (batch, hidden_size)
+        >>> cx = torch.randn(3, 20)          
         >>> output = []
-        >>> for i in range(6):
+        >>> for i in range(input.size()[0]):  
                 hx, cx = rnn(input[i], (hx, cx))
                 output.append(hx)
+        >>> output = torch.stack(output, dim = 0)
```

Comment:
nit: `dim = 0` -> `dim=0`, Python linters will complain about spacing around the equals